astropy/io/ascii/misc.py

```diff
@@ -0,0 +1,124 @@
+"""A Collection of useful miscellaneous functions.
+
+misc.py:
+  Collection of useful miscellaneous functions.
+
+:Copyright: None -- Hack away at your leisure!!
```

Comment:
@astrofrog Maybe some of these helpers are so useful that they should live in a more general `misc` and not only inside `io.ascii`? What do you think?